### ISSUE TITLE
Windows: Cannot launch gitea dump from windows service

### ISSUE THREAD

**Issue by PurdeysBane** [go-gitea/gitea#9109]
- Gitea version (or commit ref): 1.9.6
- Git version:  2.23.0.windows.1
- Operating system: Windows 10
- Database (use `[x]`):
  - [ ] PostgreSQL
  - [x] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [ ] No
  - [x] Not relevant
- Log gist:

## Description

I am trying to run gitea.exe dump -R from a Windows service. The exe appears to run but immediately returns with exit code 0 without doing anything. It does not create a log file (even if I increase the log level to trace in the app.ini file)

If I run the command line from a command prompt it works as expected. Any idea why gitea is exiting without doing anything?

**Comment by lunny** [comment]
Could you specify the config file path.

**Comment by PurdeysBane** [comment]
Thanks but I already tried that and it didn't help. 

Further information:
It also fails to run from Windows Task scheduler unless running as the current logged in user
i.e. it seems that it only works when running from an interactive session

**Comment by guillep2k** [comment]
@PurdeysBane Try running `/path/to/bash with -c "/path/to/gitea --option1 --option2"`. If command line doesn't work (Windows command prompt parser can be a b*tch), try creating a bash script:

```
#!/bin/bash

exec /path/to/gitea --option1 --option2 ...
```

And run it with `/path/to/bash /path/to/your/script`

**Comment by guillep2k** [comment]
Note: you may want to test all this from.... command line first. 😁

**Comment by rogeriorc** [comment]
Steps to reproduce:

Download PsTools (from [SysInternals)](https://docs.microsoft.com/en-us/sysinternals/downloads/psexec)

run: 

```
PsExec.exe -i -s cmd.exe
```
To open a Command Prompt as Local System Account.
Then run a dump command.
```
gitea.exe dump -V -c "D:\Gitea\custom\conf\app.ini"
```

**Comment by rogeriorc** [comment]
So...?

**Comment by lunny** [comment]
@rogeriorc which version of your gitea instance?

**Comment by rogeriorc** [comment]
Hi, @lunny !

Version 1.11.0 windows amd64

Thanks!

**Comment by guillep2k** [comment]
@rogeriorc Please notice that you cannot run `dump` while Gitea is running.

**Comment by rogeriorc** [comment]
@guillep2k the docs say:

**Backup Command (dump)** 
Switch to the user running Gitea: `su git`. Run `./gitea dump -c /path/to/app.ini` in the Gitea installation directory. There should be some output similar to the following:

It works when I'm use a user account, even the administrator account.

**Comment by guillep2k** [comment]
@rogeriorc That's right, however the `dump` command doesn't produce consistent results if the instance is running, because it's just a succession of `SELECT ... FROM ...` statements.

Anyway, be aware that we strongly discourage running Gitea as `System` (or `root` in *nix systems) for security reasons.

Back to your problem, the `Local System` account has the peculiarity of [not having a proper user profile](https://docs.microsoft.com/en-us/windows/win32/services/localsystem-account). Maybe that's what's causing your inconvenience.

**Comment by inDream** [comment]
@guillep2k @rogeriorc I think this is related to logger cannot get default file name.
It fixed after changed value for [log.file] FILE_NAME = gitea.log
